createPayload.js

```diff
@@ -1,8 +1,9 @@
 module.exports = function(name, url, element) {
-	return {
+	let returnObj = {};
+	returnObj[name] = {
```

Comment:
I think it was more straightforward the way I changed it to. The current way the object will look like:
`
[
  {
    articleLink: 'link',
    articleTitle: 'title',
    siteName: 'name,
    siteUrl: 'url'
  },
  {
    articleLink: 'link2',
    articleTitle: 'title2',
    siteName: 'name2,
    siteUrl: 'url2'
  }
]
`
Then on the front end we'll be able to get them all super quickly like:
`
result.forEach((site) => {
  console.log(site);
}
`

It will make it more dynamic. Trying something like returnObj[name] will make it so that we have to do something like Object.keys() which is kind of a pain.

I just think it will be much clearer to return an array of the objects we want, rather than an object of objects.

Reply:
I need the objects to be indexed in the database. We have a unique identifier so we should use it. We will also be separating the data into leaning on the front end so we will need to get by index anyway.